libbeat/docs/outputconfig.asciidoc

```diff
@@ -1186,3 +1211,30 @@ The following elliptic curve types are available:
 * P-256
 * P-384
 * P-521
+
+[[configuration-output-codec]]
+=== Output Codec Configuration
+
+Outputs that do not require a specific output encoding do support to change the encoding using the `codec` configuration. The codec configured must be one of `json` or `format`. By default the `json` codec is used.
+
+*`json.pretty`*: If `pretty` is set to true, events will be nicely formatted. The default is false.
+
+Example console output configuration using the `json` codec with pretty printing enabled:
+
+[source,yaml]
+------------------------------------------------------------------------------
+output.console:
+  codec.json:
+    pretty: true
+------------------------------------------------------------------------------
+
+*`format.string`*: Configurable format string used to create a custom formatted message.
+
+Example console output configuration using the `format` codec to print "--" and the events message field for every event being published:
```

Comment:
Same comment as before. I would say "Example configuration that uses the `format` codec to print each message to the console as "--" followed by the message field of the event being published.